#!/usr/bin/env python
PACKAGE = "parsian_protobuf_wrapper"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("refree_multicast_port", int_t,    0, "RefBox Multicast Port", 10003,  0, 66000)
gen.add("refree_multicast_ip"  , str_t,    0, "RefBox Multicast IP",  "224.5.23.1")


exit(gen.generate(PACKAGE, "parsian_protobuf_wrapper", "refBox"))
